main.py

```diff
@@ -27,6 +27,19 @@ class RunModes:
 
 
 def main():
+    PYTHON_BIN_ENV_VAR = "GUMTREE_PYTHON_BIN"
+    PYPARSER_ENV_VAR = "GUMTREE_PYPARSER_PATH"
+    PYTHON_BIN_VALUE = "python3"
+
+    project_directory = os.path.dirname(os.getcwd())
+    pyparser_path = project_directory + '/external/pyparser.py'
```

Comment:
I understand, that we could run it only on Linux for now, but wouldn't it be better to use `os.path.join(project_directory, 'external', 'pyparser.py')` to avoid separators like `/`?

Reply:
I am really sorry for the long answer, fixed